### ISSUE TITLE
Create data flow component

### ISSUE THREAD

**Issue by lakikowolfe** [Sage-Bionetworks/data_flow#41]
In discussion about how to store certain data flow attributes (i.e. release scheduled, embargo, etc) we realized that it does not make sense to store these attributes in the dataset manifest. Attributes related to the dataset status / flow through the pipeline from ingestion to release will be contained in this new data flow component.

The component will be updated through the release administration component within the DFA.

#### AC:

- [x] Identify attributes to store in data flow manifest
- [x] Consult with Gianna and Miley about creating a new schema / component
- [x] Create schema
- [x] Generate an example empty manifest for the Data Flow Status component
- [x] Fill in the example manifest with dummy data and submit it to a playground Project; this should store it as a Synapse  Data Flow Status (DFS) table

_Dependent on #49 and #51_ 
- [x] check if the DFA retrieves the info from the DFS table and that shows in the dashboard correctly
- [x] check if changing the status of a dataset through the DFA admin interface, submits a DFS manifest updating the Synapse DFS table
- [x] check if the DFA retrieves the updated info from the DFS table and that shows in the dashboard correctly

**Comment by milen-sage** [comment]
#45 is a dependency here.

**Comment by ychae** [comment]
@ychae @lakikowolfe @milen-sage send out RFCs to internal Sage folks for comments (July 20)

**Comment by milen-sage** [comment]
Linking [RFC](https://docs.google.com/document/d/1n8tZBdsF9dujsFszj2lUFWU2Gpl5eg1IwpSdFWg6SEA/edit#) here for ease of access.

**Comment by lakikowolfe** [comment]
[Google Sheet version of data flow component](https://docs.google.com/spreadsheets/d/18KPPBKXqUe9V6glZCPt6t6VlQ6BtjLseRrXu38IKH0Y/edit#gid=0)

Downloaded as CSV and converted to JSON LD using `schematic schema convert ~/Downloads/data_flow_component.csv`

**Comment by milen-sage** [comment]
I realize the emoji is probably an overkill used in the context of schemas, but it's neat that this is getting there :)

**Comment by lakikowolfe** [comment]
Empty manifest created using `schematic manifest -c config.yml get -o ./manifest.csv`

**Comment by lakikowolfe** [comment]
Notes:

- Might want to allow no entry for release_scheduled and embargo columns. Not every dataset will have an embargo and release might not be scheduled until after upload (through DFA administrator)
- Submitted manifest with in house validation check
```
model_submit(data_type = "DataFlow",
             dataset_id = "syn33625715",
             restrict_rules = TRUE,
             csv_file = "manifest.csv",
             input_token = input_token,
             manifest_record_type = "entity",
             url="http://localhost:3001/v1/model/submit",
             schema_url="https://raw.githubusercontent.com/Sage-Bionetworks/data_flow/implement_schema/inst/data_flow_component.jsonld")
```

**Comment by lakikowolfe** [comment]
This issue will be closed by #65
